src/components/ItemSet/index.js

```diff
@@ -80,3 +97,18 @@ export default function ItemSet({ className, setPopper, setModal, set }) {
     </div>
   );
 }
+const mapStateToProps = () => {
+  const selectedItemDefsSelector = makeSelectedItemDefsSelector();
+  return (state, ownProps) => {
+    return {
+      inventory: inventorySelector(state),
+      itemDefs: selectedItemDefsSelector(state, ownProps)
+    };
+  };
+};
```

Comment:
I don't think any of this `mapStateToProps` function is required any more? It was probably there when you started this work (and/or when the hide-sets branch was made), but its no longer needed.


I think that argument is optional, so you could just remove it entirely (have only `connect(mapDispatchToActions)`). Otherwise, just inline an function that returns empty object (`connect(() => ({}), mapDispatchToActions)`)